### PR TITLE
fix to java w/ version update and update to aae/pb tests for RIAK-1557 i...

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
...nvolving entropy data and spaces in types/buckets/keys.

This fixes up the issue w/ Entropy Data described in https://github.com/basho/yokozuna/issues/450 and https://github.com/basho/yokozuna/issues/436. 
- [x] update version #'s in yz jars within https://github.com/basho/yokozuna/blob/develop/tools/build-jar.sh and https://github.com/basho/yokozuna/blob/develop/tools/grab-solr.sh and upload to s3.

### CODE REVIEW

**Comment by zeeshanlakhani** [comment]
@seancribbs I uploaded yokozuna-2.jar and yokozuna-2.jar.sha to S3.

File: misc/bench/src/yz_driver.erl [hunk 1]
```diff
@@ -9,6 +9,7 @@
 -include_lib("basho_bench/include/basho_bench.hrl").
 -record(state, {default_field, fruits, pb_conns, index, bucket, iurls, surls}).
 -define(DONT_VERIFY, dont_verify).
+-define(SPACER, 'foo ').
```
Comment (bearcage) [inline misc/bench/src/yz_driver.erl]: Total nitpick, but mind switching the prefix on space testing keys to something that makes the purpose clear if you're tracking down a problem later?

File: misc/bench/src/yz_driver.erl [hunk 2]
```diff
@@ -159,6 +160,17 @@ run(load_fruit, KeyValGen, _, S=#state{iurls=URLs}) ->
         {error, Reason} -> {error, Reason, S2}
     end;
 
+run(load_fruit_plus_spaces, KeyValGen, _, S=#state{iurls=URLs}) ->
+    Base = get_base(URLs),
+    {Key, Val} = KeyValGen(),
+    Key2 = mochiweb_util:quote_plus(lists:concat([?SPACER, Key])),
```
Comment (seancribbs) [inline misc/bench/src/yz_driver.erl:166]: Does this even work? `?SPACER` is an atom.

Comment (zeeshanlakhani) [inline misc/bench/src/yz_driver.erl:166]: It does... but should I just make it a string?

Comment (zeeshanlakhani) [inline misc/bench/src/yz_driver.erl:166]:
``` erlang
 list_to_binary(lists:concat([' foo bar', binary_to_integer(<<"44">>)])).
```

would then return `<<" foo bar44">>`... which would get formatted correctly on the url calls.

Comment (zeeshanlakhani) [inline misc/bench/src/yz_driver.erl:166]: string works too... so whichever is preferred :D.

Comment (seancribbs) [inline misc/bench/src/yz_driver.erl:166]: I think a string would be less surprising, that's all.

Comment (zeeshanlakhani) [inline misc/bench/src/yz_driver.erl:166]: :+1:

**Comment by seancribbs** [comment]
:+1: ca768f3

```
Test Results:
aae_test-bitcask: pass
---------------------------------------------
0 Tests Failed
1 Tests Passed
That's 100.0% for those keeping score
```

**Comment by zeeshanlakhani** [comment]
@borshop merge

**Pull request merged**